index.html

```diff
@@ -1,311 +1,321 @@
-            <ul class="icon-lang">
-              <li>
-                <span class="icon-section"><img src="/img/js.png" alt="icon-js"> <span
-                    class="javascript">JavaScript</span></span>
-              </li>
-              <li>
-                <span class="icon-section"><img src="/img/ic-css.png" alt="icon-css"> <span
-                    class="css-lang">CSS</span></span>
-              </li>
-              <li>
-                <span class="icon-section"><img src="/img/html.png" alt="icon-html"> <span
-                    class="httml">HTML</span></span>
-              </li>
-            </ul>
-          </li>
-          <li class="fram">
-            Frameworks
-            <img class="down-icon" src="/img/downicon.png" alt="Drop Down icon">
-          </li>
-          <li class="fram">
-            Skills
-            <img class="down-icon" src="/img/downicon.png" alt="Drop Down icon">
-          </li>
-
-        </ul>
-      </div>
-    </div>
-
-  </section>
-
-  <section class="contact-form">
-     <div class="contact-content">
-      <h1 class="contact-me">Contact me</h1>
-      <p class="contact">
-        If you have an application you are interested in developing,
-         a feature that you need built or a project that needs coding.
-          I’d love to help with it </p>
-    </div>
-    <form action="https://formspree.io/f/xqkjaqoe" method="post">
-    <div class="name-container">
-      <input name="name" type="text" placeholder="Yeremias" maxlength="30" required>
-    </div>
-    <br>
-    <div class="mail-container">
-      <input name="email" type="email" placeholder="natajayanj14@gmail.com" required>
-    </div>
-    <br>
-    <div class="textarea-container">
-      <textarea name="message" maxlength="500" placeholder="Write your message here" required></textarea>
-    </div>
-    <br>
-    <button type="submit">Get it Touch</button>
-  </form>
-   
-    
-  </section>
+    <header>
+        <a class="logo" href="#">Abdo</a>
+        <a href="#" class="menu">menu</a>
+        <nav class="navigation">
+            <a class="links" href="#portfolio">Portfolio</a>
+            <a class="links" href="#about">About</a>
+            <a class="links" href="#contact">Contact</a>
+        </nav>
+    </header>
+    <main class="wrapper">
 
+        <section class="intro">
+            <div class="container-global">
+                <div class="title description">
+                    <h1 class="title">I’m Abdo<br>
+                        Glad to see you!</h1>
+                    <p class="description">I’m a software developer! I can help you build a product , feature or website
+                        Look through some of my work and experience! If you like what you see and have a project you
+                        need coded, don’t hestiate to contact me.</p>
+
+                </div>
+                <div class="social container">
+                    <a href="#" class="button connect">LET’S CONNECT</a>
```

Comment:
- [OPTIONAL] Kindly align this text to the left as in the Figma template. Stick to the Figma template is a project requirement. As not required by the previous reviewer I consider this optional.